specification/monitor/Azure.Monitor.Ingestion/routes.tsp

```diff
@@ -29,6 +31,8 @@ op upload is Azure.Core.Foundations.Operation<
 
     /** The array of objects matching the schema defined by the provided stream. */
     #suppress "@azure-tools/typespec-azure-core/request-body-problem"
+    @alternateType(unknown, "go")
+    @clientName("logs", "go")
```

Comment:
Is this required? `client.tsp` already overrides this operation and adds these decorators.